index.d.ts

```diff
@@ -68,3 +67,32 @@ export default function elementReady<ElementName extends Element = HTMLElement>(
 	selector: string,
 	options?: Options
 ): StoppablePromise<ElementName | undefined>;
+
+/**
+Detect when elements are ready in the DOM. Returns an async iterator which yields with each new matching element. Useful for user-scripts that modify elements when they are added.
```

Comment:
```suggestion
Detect when elements are ready in the DOM.

Useful for user-scripts that modify elements when they are added.
```

The return sentence is already below.